Sources/Basics/FileSystem+Extensions.swift

```diff
@@ -238,13 +238,16 @@ extension FileSystem {
 
 extension FileSystem {
     public func stripFirstLevel(of path: AbsolutePath) throws {
-        let topLevelContents = try self.getDirectoryContents(path)
-        guard topLevelContents.count == 1, let rootPath = topLevelContents.first.map({ path.appending(component: $0) }), self.isDirectory(rootPath) else {
+        let topLevelDirectories = try self.getDirectoryContents(path)
+            .map{ path.appending(component: $0) }
+            .filter{ self.isDirectory($0) }
```

Comment:
I am a bit on-the-fence about whether to filter out all files or just files that start with a `.` (all temporary files tend to have that prefix), but I think the new heuristic is good in that it allows an extra README etc.  So I think this is good.